app/server/src/client/preview/render.ts

```diff
@@ -6,9 +6,9 @@ import { RenderContext, FetchStoryHtmlType } from './types';
 
 const rootElement = document.getElementById('root');
 
-const defaultFetchStoryHtml: FetchStoryHtmlType = async (url, path, params) => {
+const defaultFetchStoryHtml: FetchStoryHtmlType = async (url, path, params, storyContext) => {
   const fetchUrl = new URL(`${url}/${path}`);
-  fetchUrl.search = new URLSearchParams(params).toString();
+  fetchUrl.search = new URLSearchParams({ ...params, ...storyContext.globals }).toString();
```

Comment:
I still wonder if we should have a configuration setting to control this behavior. Should it be opt in?

Reply:
Opt-in as in configuration parameter to merge the globals? Another option is to not merge globals and let the user write a custom `fetchStoryHtml` if they need globals.

Reply:
Yes opt-in to merge. We could force the user to write a custom `fetchStoryHtml` but as seen here in the file there is enough going on that its a pretty heavy lift it all you want is globals. 

Maybe I'm over thinking the automatic merge of globals. It's unlikely that a few extra query params would break the  server implementation, they'd get ignored. Perhaps we stick with this implementation and wait for someone make the feature request for something more complicated and a real world usecase to solve against?

Reply:
Do we worry about conflicts between globals and params keys? Should we combine in the opposite order so that params would take precedence in any conflict?

Reply:
Yes swaping the order makes sense. I'll change that.

I think it makes sense to have them by default. I would say that, as a user, if you use globalTypes you would expect to receive them in the query (at least I did).